src/libs/ReportPrimaryActionUtils.ts

```diff
@@ -0,0 +1,168 @@
+import type {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import type {Policy, Report, Transaction, TransactionViolation} from '@src/types/onyx';
+import {isApprover as isApprovedMember} from './actions/Policy/Member';
+import {getCurrentUserAccountID} from './actions/Report';
+import {hasAccountingConnections, isAutoSyncEnabled, isPrefferedExporter} from './PolicyUtils';
+import {
+    isClosedReport,
+    isCurrentUserSubmitter,
+    isExpenseReport,
+    isHoldCreator,
+    isInvoiceReport,
+    isIOUReport,
+    isOpenReport,
+    isPayer,
+    isProcessingReport,
+    isReportApproved,
+} from './ReportUtils';
+import {getSession} from './SessionUtils';
+import {isDuplicate, isOnHold as isOnHoldTransactionUtils} from './TransactionUtils';
+
+function isSubmitAction(report: Report) {
+    const isExpense = isExpenseReport(report);
+    const isSubmitter = isCurrentUserSubmitter(report.reportID);
+    const isOpen = isOpenReport(report); 
+    const isManualSubmitEnabled = true; // TODO how to find it?
```

Comment:
@luacmartins I'm not sure how this value should be calculated and I haven't found something similar in codebase.

Reply:
@jnowakow you can do something like this https://github.com/Expensify/App/blob/6410259e9096d4026f1c3e45b2c0c94b81223d20/src/libs/actions/Policy/Policy.ts#L446